test/index.js

```diff
@@ -0,0 +1,16 @@
+var expect = require('chai').expect
+var cp = require('child_process')
+
+describe('main.js', function () {
+  it('must print green, red and yellow logs', function(done){
+    child = cp.exec('node main', function (error, output, errorOutput) {
```

Comment:
I think it's overkill to spawn a child process here. Why not test the module directly?